src/js/poster-image.js

```diff
@@ -72,8 +79,11 @@ class PosterImage extends Button {
   }
 
   /**
-   * Set the poster source depending on the display method
-   */
+  * Set the poster source depending on the display method
```

Comment:
The alignment of this one looks like it needs to be one space to the right. Not sure if that actually affects parsing.